cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -154,7 +154,7 @@ func TestGetAPIServerCommand(t *testing.T) {
 			},
 			expected: []string{
 				"kube-apiserver",
-				"--insecure-port=0",
+				"--secure-port=fmt.Sprintf("%d", cfg.LocalAPIEndpoint.BindPort)",
```

Comment:
`Sprintf` is a number of times slower than `Itoa`, please use that and also ideally store it in a variable per function.

i think it's OK to change the instances in `manifests.go` too.